src/schemas/FEEDBACK-TOOL/schema.js

```diff
@@ -67,12 +72,17 @@ const internationalSchoolAddress = {
     street: {
       type: 'string',
       minLength: 1,
-      maxLength: 80
+      maxLength: 53
     },
     street2: {
       type: 'string',
       minLength: 1,
-      maxLength: 80
+      maxLength: 53
+    },
+    street3: {
```

Comment:
Depending on how we combine these on the backend, we may want to reduce these to 52, to allow for an additional ", " (comma and space), between street2 and street3.